resources/css/legacy.css

```diff
@@ -3,8 +3,9 @@
   padding: 0;
   position: relative;
   min-height: 450px;
-  display: flex;
-  grid-gap: 1rem;
+  display: grid;
+  column-gap: 1rem;
+  grid-template-columns: 1.329fr 0.671fr;
```

Comment:
It used to be grid, but caused trouble when widening the overall container width.
While content shift/SEO is important, this site is not quite in a state where this is of the highest priority imo.